documentation/ipi-install/modules/ipi-install-config-yaml-4.4.adoc

```diff
@@ -1,66 +1,66 @@
 apiVersion: v1
-baseDomain: <domain>
+xref:basedomain[basedomain]: +<domain>+
 metadata:
-  name: <cluster-name>
+  xref:metadataname[name]: +<cluster-name>+
 networking:
-  machineCIDR: <public-cidr>
+  xref:machinecidr[machineCIDR]: +<public-cidr>+
   networkType: OVNKubernetes
 compute:
-- name: worker
-  replicas: 2
+- xref:workername[name]: worker
+  xref:computereplicas[replicas]: 2
 controlPlane:
-  name: master
-  replicas: 3
+  xref:controlplanename[name]: master
+  xref:controlplanereplicas[replicas]: 3
   platform:
-    baremetal: {}
+    xref:baremetalplatform[baremetal]: {}
 platform:
   baremetal:
-    apiVIP: <api-ip>
-    ingressVIP: <wildcard-ip>
-    dnsVIP: <dns-ip>
-    provisioningNetworkInterface: <NIC1>
-    provisioningNetworkCIDR: <CIDR> # i.e. fd00:1101::1/64
-    provisioningBridge: provisioning
-    hosts:
+    xref:apivip[apiVIP]: +<api-ip>+
+    xref:ingressvip[ingressVIP]: +<wildcard-ip>+
+    dnsVIP: +<dns-ip>+
+    provisioningNetworkInterface: +<NIC1>+
+    provisioningNetworkCIDR: +<CIDR>+ # i.e. fd00:1101::1/64
+    #provisioningBridge: provisioning
+    xref:hoststable[hosts]:
       - name: openshift-master-0
         role: master
```

Comment:
Since we are doing xrefs for all parameters, we should include these as well. I won't recomment on the others as theres quite a few.

Reply:
Pushed